libs/application/templates/general-petition/src/lib/generalPetitionApplicationTemplate.ts

```diff
@@ -73,6 +73,14 @@ const GeneralPetitionApplicationTemplate: ApplicationTemplate<
                 ),
               read: 'all',
             },
+            {
+              id: Roles.SIGNATUREE,
+              formLoader: () =>
+                import('../forms/EndorsementForm').then((val) =>
+                  Promise.resolve(val.EndorsementForm),
+                ),
+              read: 'all',
```

Comment:
Is there anything sensitive here which we don't want the signaturee to have access to?